UnitTests/GitCommands.Tests/Git/ExecutableExtensionsTests.cs

```diff
@@ -111,7 +111,7 @@ public void GetOutput_with_cache_miss()
                 Assert.AreEqual(expectedProcessedCounts[index], eventArgs.ProcessedCount);
                 index++;
             });
-            Assert.IsTrue(result.Value.ExitedSuccessfully);
+            Assert.AreEqual(result.Value.ExitCode, 0);
```

Comment:
🤔 I think these are equivalent.
```
public bool ExitedSuccessfully => ExitCode == 0;
```

Reply:
The tests run in the GE directory, so checking the output is not a good test...
The test make a reset, it could reset files too (but there are no GE files that could match the random name I believe).
These tests should run in a dedicated repo. Separate PR.